api/kroxylicious-api/pom.xml

```diff
@@ -26,19 +26,20 @@
     <description>generic APIs used throughout Kroxylicious</description>
 
     <dependencies>
+        <!-- third party dependencies - runtime and compile -->
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-annotations</artifactId>
         </dependency>
-        <!-- Test dependencies -->
+        <!-- third party dependencies - test -->
         <dependency>
-            <groupId>org.junit.jupiter</groupId>
-            <artifactId>junit-jupiter</artifactId>
+            <groupId>org.assertj</groupId>
```

Comment:
I'm guessing you've alphabetised the dependencies, did you find a tool to do it or is this done by hand?

As I'm only interested in doing it if its automated otherwise its going to bit rot.